Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -744,7 +744,7 @@ private BuildLog Execute_Settings_ValidSetup(bool isTestProject, string msBuildL
                     }
             };
 
-            // Create the project
+            // Create the project. 'sonar.cs.roslyn.ignoreIssues' is 'true' by default. Additional analyzers will be removed.
```

Comment:
`sonar.cs.roslyn.ignoreIssues` "Defaults to false."  ([docs](https://docs.sonarqube.org/latest/analysis/external-issues/)) - has this changed in this PR? why?

Reply:
This is a good catch of an old problem. 

From Scanner and UT point of view, default is `true`, see current master, line 182
https://github.com/SonarSource/sonar-scanner-msbuild/blob/0779f4d58eec352021ce36e32fe8df574c78bba8/src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs#L180-L184

But it accidentally works, because server always provides explicit value.

I think it's a mistake from 2018 and this commit https://github.com/SonarSource/sonar-scanner-msbuild/commit/6ae4c5e0 where `sonar.{language}.roslyn.importAllIssues` was changed to `sonar.{language}.roslyn.ignoreIssues` with inverted meaning, but the default `true` value remained the same.

Let's fix this in another PR, bunch of UTs will need to be updated I guess.

cc @mickael-caro-sonarsource

Reply:
Fix is on it's way in #990